File: lang/csharp/src/apache/main/CodeGen/AIngleRuntimeException.cs
/*

 */
using System;

namespace AIngle
{
    /// <summary>
    /// A generic AIngle exception.
    /// </summary>
    public class AIngleRuntimeException : AIngleException
    {
        /// <summary>
        /// Initializes a new instance of the <see cref="AIngleRuntimeException"/> class.
        /// </summary>
        public AIngleRuntimeException()
        {
        }

        /// <summary>
        /// Initializes a new instance of the <see cref="AIngleRuntimeException"/> class.
        /// </summary>
        /// <param name="s">The message that describes the error.</param>
        public AIngleRuntimeException(string s)
            : base(s)
        {
        }

        /// <summary>
        /// Initializes a new instance of the <see cref="AIngleRuntimeException"/> class.
        /// </summary>
        /// <param name="s">The message that describes the error.</param>
        /// <param name="inner">
        /// The exception that is the cause of the current exception, or a null reference
        /// if no inner exception is specified.
        /// </param>
        public AIngleRuntimeException(string s, Exception inner)
            : base(s, inner)
        {
        }
    }
}

File: lang/java/aingle/src/main/java/org/apache/aingle/AIngleRuntimeException.java
/*

 */

package org.apache.aingle;

/** Base AIngle exception. */
public class AIngleRuntimeException extends RuntimeException {
  public AIngleRuntimeException(Throwable cause) {
    super(cause);
  }

  public AIngleRuntimeException(String message) {
    super(message);
  }

  public AIngleRuntimeException(String message, Throwable cause) {
    super(message, cause);
  }
}
